### PR TITLE
Fix warnings on 1.6

### PR DESCRIPTION
**Pull request by phaseinducer**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - "2.7"
+
+sudo: false
```
Comment (mickaelandrieu) [inline .travis.yml]: <3

**Comment by mickaelandrieu** [comment]
GTM! Thanks @phaseinducer :)

File: cookbook/mass_edition/register_a_new_mass_edit_action.rst [hunk 2]
```diff
@@ -183,15 +183,17 @@ Phase 6: add user groups permissions to job profiles (ENTERPRISE EDITION)
...
+execute an insert SQL query to add these permissions.
+It will be something like:
     
 .. code-block:: mySQL
 
-INSERT INTO pimee_security_job_profile_access VALUES ('', <job_profile_id>, <user_group_id>, 1, 1);
+    INSERT INTO pimee_security_job_profile_access VALUES ('', <job_profile_id>, <user_group_id>, 1, 1);
 
-The two last number means you give respectively 'edit' and 'execution' permissions. Otherwise add '0'.
+The two last numbers means you give respectively 'edit' and 'execution' permissions. Otherwise add '0'.
```
Comment (rybus) [inline cookbook/mass_edition/register_a_new_mass_edit_action.rst]: mean

File: reference/import_export/product-export.rst [hunk 3]
```diff
@@ -142,7 +142,7 @@ The service is defined in ``src/Pim/Bundle/ConnectorBundle/Resources/config/proc
 
 The class ``Pim\Component\Connector\Processor\Normalization\ProductProcessor`` mainly delegates the transformation to the service ``pim_serializer.normalizer.product``.
 
-We can see here that we normalize each product into the ``standard`` format. It is the writer's responsibility to convert the standard format to the flat format. (cf :doc:`/cookbook/import_export/clean-csv-file-during-product-import.rst`)
+We can see here that we normalize each product into the ``standard`` format. It is the writer's responsibility to convert the standard format to the flat format. (cf :doc:`/cookbook/import_export/clean-csv-file-during-product-import`)
```
Comment (rybus) [inline reference/import_export/product-export.rst]: it is the responsability of the writer

**Comment by rybus** [comment]
Few comments then GTM :)

**Pull request merged**